assets/js/modules/idea-hub/components/dashboard/DashboardCTA/index.stories.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Idea Hub DashboardCTA component stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createTestRegistry, WithTestRegistry, provideModules } from '../../../../../../../tests/js/utils';
+import DashboardCTA from './index';
+
+const Template = ( args ) => <DashboardCTA { ...args } />;
+
+export const DefaultDashboardCTA = Template.bind( {} );
+DefaultDashboardCTA.storyName = 'Default';
+DefaultDashboardCTA.decorators = [
+	( Story ) => {
+		const registry = createTestRegistry();
+		provideModules( registry, [ {
+			slug: 'idea-hub',
+			active: false,
+			connected: false,
+		} ] );
+
+		return (
+			<WithTestRegistry registry={ registry } features={ [ 'ideaHubModule' ] }>
```

Comment:
This only works because it doesn't rely on any thing registered from its datastore/module. When the test registry is created above, the `idea-hub` feature won't be enabled and so this may lead to some odd behavior/results.

This should be fixed in #2940 which is just pending review, but would require slightly different implementation here.

Reply:
@aaemnnosttv Could you elaborate on this please? 

> When the test registry is created above, the idea-hub feature won't be enabled and so this may lead to some odd behavior/results.

This CTA is rendered when the feature is not enabled... maybe I am misunderstanding something. Should I be using `enabledFeatures.clear();` to ensure that it is not enabled, for example?

> This should be fixed in #2940 which is just pending review, but would require slightly different implementation here.

I'm looking through the changes in [Asvin's PR](https://github.com/google/site-kit-wp/pull/3382/files/8a453ed75bc2ddd7066eaa535a8d77e859c3e34f# ) which got merged and seems to be fairly similar. 
Am I ok to use that as a guide for this? Is there anything specific I need to include?

Reply:
@johnPhillips #2940 is merged now so the key takeaways there are:

- Stories should no longer be wrapped with `WithTestRegistry` _in the story itself_ as this is applied in a global decorator now to every story
    - Features should be set on `Story.parameters.features` now instead of passing as a prop to the test registry
    - All features are be disabled by default for each story automatically, so nothing extra is needed for a story that needs a disabled feature
- Similarly, `createTestRegistry` shouldn't be used at the story level anymore – maybe there's some edge case where it's needed, but as a general rule we only need to provide registry setup for the story (see `WithRegistrySetup`)


The PR you referenced there isn't quite up to date with all of the above. Best to reference `DashboardSearchVisitorsWidget.stories.js` first and then let me know if you still have questions